documentation/asciidoc/computers/camera/rpicam_options_common.adoc

```diff
@@ -44,533 +48,515 @@ Available cameras
+* `<h>`: image height to extract
 
-----
-	--roi				Select a crop (region of interest) from the camera <x,y,w,h>
-----
+Defaults to `0,0,1,1` (starts at the first X coordinate and the first Y coordinate, uses 100% of the image width, uses 100% of the image height).
 
-The `--roi` (region of interest) option allows the user to select a particular crop from the full field of view provided by the sensor. The coordinates are specified as a proportion of the available field of view, so `--roi 0,0,1,1` would have no effect at all.
+Examples:
 
-The `--roi` parameter implements what is commonly referred to as "digital zoom".
+* `rpicam-hello --roi 0.25,0.25,0.5,0.5` selects exactly a half of the total number of pixels cropped from the centre of the image (skips the first 25% of X coordinates, skips the first 25% of Y coordinates, uses 50% of the total image width, uses 50% of the total image height).
+* `rpicam-hello --roi 0,0,0.25,0.25` selects exactly a quarter of the total number of pixels cropped from the top left of the image (skips the first 0% of X coordinates, skips the first 0% of Y coordinates, uses 25% of the image width, uses 25% of the image height).
 
-Example `rpicam-hello --roi 0.25,0.25,0.5,0.5` will select exactly a quarter of the total number of pixels cropped from the centre of the image.
+==== `hdr`
 
-----
-	--hdr				Run the camera in HDR mode <mode>
-----
+Default value: `off`
 
-The `--hdr` option causes the camera to be run in the HDR (High Dynamic Range) mode given by `<mode>`. On Pi 4 and earlier devices, this option only works for certain supported cameras, including the _Raspberry Pi Camera Module 3_, and on Pi 5 devices it can be used with all cameras. `<mode>` may take the following values:
+Runs the camera in HDR mode. If passed without a value, assumes `auto`. Accepts one of the following values:
 
-* `off` - HDR is disabled. This is the default value if the `--hdr` option is omitted entirely.
-* `auto` - If the sensor supports HDR, then the on-sensor HDR mode is enabled. Otherwise, on Pi 5 devices, the Pi 5's on-chip HDR mode will be enabled. On a Pi 4 or earlier device, HDR will be disabled if the sensor does not support it. This mode will be applied if the `--hdr` option is supplied without a `<mode>` value.
-* `single-exp` - On a Pi 5, the on-chip HDR mode will be enabled, even if the sensor itself supports HDR. On earlier devices, HDR (even on-sensor HDR) will be disabled.
+* `off` - Disables HDR.
+* `auto` - Enables HDR on supported devices. Uses the sensor's built-in HDR mode if available. If the sensor lacks a built-in HDR mode, uses on-board HDR mode, if available.
+* `single-exp` - Uses on-board HDR mode, if available, even if the sensor has a built-in HDR mode. If on-board HDR mode is not available, disables HDR.
 
-Example: `rpicam-still --hdr -o hdr.jpg` for capturing a still image, or `rpicam-vid --hdr -o hdr.h264` to capture a video.
+Raspberry Pi 5 and later devices have an on-board HDR mode.
```

Comment:
IMHO the original text of "on-chip" might be slightly clearer here, because "on-board" feels like a much looser term?
"On-board" could mean _somewhere_ on the Pi5's PCB, or _somewhere_ on the Camera's PCB, whereas "on-chip" feels much more obviously something to do with BCM2712 or RP1.

OTOH perhaps that's a level of distinction that non-engineers don't need? :shrug: :confused: